src/database/entities/NotablePersonEvent.ts

```diff
@@ -34,7 +34,7 @@ export class NotablePersonEvent extends BaseEntity {
 
   @ValidateIf((_, v) => typeof v === 'string')
   @IsNotEmpty()
-  @Column({ type: 'text', nullable: true })
+  @Column({ type: 'varchar', length: 1000, nullable: true, unique: true })
```

Comment:
I think this is okay. 1000 characters is probably plenty enough for a quote. But I'm curious why you made this change?

Also, are we saying no two NotablePersons can say the same words?

Reply:
The 1000 chars limit is required to create the index (limitation of MySQL).

Yeah, I think the main usage of this would be to prevent double-posting. A better approach would be to create a composite index of (id, quote).